Library/Formula/algernon.rb

```diff
@@ -0,0 +1,26 @@
+class Algernon < Formula
+  desc "HTTP/2 web server with Lua support"
+  homepage "http://algernon.roboticoverlords.org/"
+  url "https://github.com/xyproto/algernon/archive/0.74.tar.gz"
+  version "0.74"
```

Comment:
You're fine to remove this line. It's simple enough that `brew` can determine it from the URL.